.gitignore

```diff
@@ -133,3 +133,7 @@ dmypy.json
 
 scripts/minio-conf/
 scripts/minio-data/
+
+# Editor/Local
```

Comment:
I guess, you can make use of personal git ignore rules?
For reference - https://www.atlassian.com/git/tutorials/saving-changes/gitignore#personal-git-ignore-rules

Reply:
Personal git configs :ok_hand: Actually thanks for the tip :raised_hands: 

I feel we can still have these in our .gitignore just in case someone is new to git and makes their first contribution on this repo.

This way a lot of junk won't be able to make it through.

Reply:
Okay, @saltgen! 
Will it be possible for you to raise a separate PR for gitignore change? as these changes are not relevant to original task. 
Regardless of how big or tiny change it is, I would prefer to open a separate pull-request. In my opinion it is easy to move such change quickly rather than blocking it due to original task.

Reply:
Yes I agree, sounds good!
I will add in a few more exclusions for the ignore file on the new PR.

Reply:
Done! PR [here](https://github.com/RedHatInsights/ros-backend/pull/127)